pagerduty/event_orchestration_cache_variable.go

```diff
@@ -0,0 +1,119 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+)
+
+type EventOrchestrationCacheVariableService service
+
+type EventOrchestrationCacheVariableCondition struct {
+	// A PCL string: https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview
+	Expression string `json:"expression,omitempty"`
+}
+
+// Configuration for a cache variable changes depending on the type:
+//   - if `Type` is `recent_value`; then use `Regex` and `Source`
+//   - if `Type` is `trigger_event_count`; then use `TTLSeconds`
+type EventOrchestrationCacheVariableConfiguration struct {
+	Type       string `json:"type,omitempty"`
+	Regex      string `json:"regex,omitempty"`
+	Source     string `json:"source,omitempty"`
+	TTLSeconds int    `json:"ttl_seconds,omitempty"`
+}
+
+// A reference to a related object (e.g. an User, etc)
+type EventOrchestrationCacheVariableReference struct {
+	ID   string `json:"id,omitempty"`
+	Type string `json:"type,omitempty"`
+	Self string `json:"self,omitempty"`
+}
+
+type EventOrchestrationCacheVariable struct {
+	ID            string                                   			`json:"id,omitempty"`
+	Name          string                                    		`json:"name,omitempty"`
+	Disabled      bool 																          `json:"disabled,omitempty"`
+	Conditions    []*EventOrchestrationCacheVariableCondition   `json:"conditions,omitempty"`
```

Comment:
And same with `Conditions` - because we want to be able to send empty arrays we should remove `omitempty`.